containers/haskell/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use the [Choice] comment to indicate option arguments that should appear in VS Code UX
+#
+# [Choice] Haskell version: 8,
+ARG VARIANT=8
+FROM haskell:${VARIANT}
+
+# Use the [Option] comment to specify true/false arguments that should appear in VS Code UX
+#
+# [Option] Install zsh
+ARG INSTALL_ZSH="true"
+# [Option] Upgrade OS packages to their latest versions
+ARG UPGRADE_PACKAGES="false"
+
+# Install needed packages and setup non-root user. Use a separate RUN statement to add your own dependencies
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+COPY library-scripts/*.sh /tmp/library-scripts/
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && /bin/bash /tmp/library-scripts/common-debian.sh "${INSTALL_ZSH}" "${USERNAME}" "${USER_UID}" "${USER_GID}" "${UPGRADE_PACKAGES}" \
+    #
+    # ****************************************************************************
+    # * TODO: Add any additional OS packages you want included in the definition *
+    # * here. We want to do this before cleanup to keep the "layer" small.
+    # ****************************************************************************
+    # && apt-get -y install --no-install-recommends <your-package-list-here> \
+    #
```

Comment:
This is in the template to let you know where you can add packages.  It can be removed for the actual definition.

```suggestion
```